common4j/src/main/com/microsoft/identity/common/java/crypto/key/ISecretKeyLoader.java

```diff
@@ -0,0 +1,25 @@
+package com.microsoft.identity.common.java.crypto.key;
+
+import com.microsoft.identity.common.java.exception.ClientException;
+
+import javax.crypto.SecretKey;
+
+import lombok.NonNull;
+
+public interface ISecretKeyLoader {
```

Comment:
Overall comment - it seems like we could hide the SecretKey implementation here.  Should we just do that?

Reply:
Also, there don't seem to be a lot of uses of this interface....

Reply:
Moved the SecretKey stuffs in AES256KeyLoader here.